routes/baseDBMethods.js

```diff
@@ -20,15 +22,48 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function processPaginationParameters(ctx) {
+  let pageNumber = parseInt(ctx.query.page, 10);
+  if (Number.isNaN(pageNumber)) {
+    pageNumber = DEFAULT_PAGE_NUMBER;
+  } else if (pageNumber < 1) {
+    throw new RangeError('Query parameter \'page\' must be a positive number');
+  }
+
+  let pageSize = parseInt(ctx.query.page_size, 10);
+  if (Number.isNaN(pageSize)) {
+    pageSize = DEFAULT_PAGE_SIZE;
+  } else if (pageSize < 1) {
+    throw new RangeError('Query parameter \'page_size\' must be a positive number');
+  }
+  pageSize = Math.min(MAX_ALLOWED_PAGE_SIZE, pageSize);
```

Comment:
If pageSize < 1 what the function will do? Will it get there?

Reply:
I am talking about Math.min

Reply:
It won't get there, since it throws an error.